cmd/solarwinds-otel-collector/go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/solarwinds/solarwinds-otel-collector
 
-go 1.22.7
-
-toolchain go1.22.9
+go 1.23.4
```

Comment:
is this required due to transitive imports from the extension? So this can't run on lower version?